smriprep/workflows/anatomical.py

```diff
@@ -222,15 +222,11 @@ def init_anat_preproc_wf(
     """
     workflow = Workflow(name=name)
     num_t1w = len(t1w)
-    desc = """
+    desc = f"""
 Anatomical data preprocessing
 
-: """
-    desc += """\
-A total of {num_t1w} T1-weighted (T1w) images were found within the input
-BIDS dataset.""".format(
-        num_t1w=num_t1w
-    )
+: A total of {num_t1w} T1-weighted (T1w) images were found within the input
```

Comment:
The resulting string `desc` is:
```
Anatomical data preprocessing

: A total of num_t1w T1-weighted (T1w) images were found within the input
BIDS dataset.
```
Is this really what we want?

Reply:
@effigies What about this one? Was on purpose?

Reply:
I think this is a misreading. If you check https://output.circle-artifacts.com/output/job/c1a05c82-b211-4baa-88e9-2b93fcd61dbc/artifacts/0/bids-run/sub-01.html#boilerplate, we do interpolate the number.

Reply:
I was referring to the leading **': '** at the start of the paragraph. The Mardown outptut looks like this:
```txt
Anatomical data preprocessing

: A total of 1 T1-weighted (T1w) images were found within the input
BIDS dataset.The T1-weighted (T1w) image was corrected for intensity non-uniformity (INU)
```
and as far as I know **': '** does not have a special meaning, and is rendered:
```markdown
Anatomical data preprocessing

: A total of 1 T1-weighted (T1w) images were found within the input
BIDS dataset.The T1-weighted (T1w) image was corrected for intensity non-uniformity (INU)
```

Reply:
I think technically it's using [https://garrettgman.github.io/rmarkdown/authoring_pandoc_markdown.html#definition-lists](definition lists) to make "Anatomical data processing" into almost-but-not-quite a section heading.

Reply:
That was the https://markdownlivepreview.com/ renderer that doesn't understand this Pandoc extension.

Using https://dillinger.io, the Markdown output looks slightly better, but still not perfect (the newline is part of the rendereding):

![markdown](https://github.com/nipreps/smriprep/assets/3234522/012564f8-376a-459b-bbad-ec91ac1faa43)

Is the Markdown output just for conversion with Pandoc, or is it supposed to be usable as such?